roxie/ccd/ccdserver.cpp

```diff
@@ -719,16 +714,13 @@ class CRoxieServerActivityFactory : public CRoxieServerActivityFactoryBase
 class CRoxieServerMultiOutputFactory : public CRoxieServerActivityFactory
 {
 protected:
-    unsigned numOutputs;
-    unsigned __int64 *processedArray;
-    bool *startedArray;
+    unsigned numOutputs = 0;
+    std::atomic<__int64> *processedArray = nullptr;
```

Comment:
Should this be unsigned __int64?

Reply:
Technically unsigned is more correct - I used signed for consistency with the processed variable (see changes above). I could have changed that to unsigned alternatively (in reality it's not going to make a difference any time before the heat-death of the universe, I think)